main/utils/recipient.py

```diff
@@ -0,0 +1,30 @@
+from main.models import Recipient
+from django.db.models import Q
+
+class Recipient(object):
```

Comment:
This class name conflicts with the Recipient model name. Please rename this class to `RecipientHandler` or something else.

Reply:
8677b4da30eddf4066bb5e5650e130b0201f4259  @joemarct  done :+1: